### PR TITLE
Tyntec/wa rich media

### PR DESCRIPTION
**Pull request by ridlees**
Updates responses

### CODE REVIEW

File: certified-connectors/tyntec-WA/apiDefinition.swagger.json [hunk 1]
```diff
@@ -302,26 +302,36 @@
 		"/chat-api/v2/messages/": {
 			"post": {
 				"responses": {
-					"default": {
-						"description": "default",
-						"schema": {}
+					"202": {
```
Comment (sriyen-msft) [inline certified-connectors/tyntec-WA/apiDefinition.swagger.json:305]: @ridlees Indentations - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is longer tab index between the properties. Please refer the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/tyntec-WA/apiDefinition.swagger.json (hunk 1 above)
Comment (sriyen-msft) [reply certified-connectors/tyntec-WA/apiDefinition.swagger.json:305]: @ridlees Do you have any update on this?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/tyntec-WA/apiDefinition.swagger.json (hunk 1 above)
Comment (ridlees) [reply certified-connectors/tyntec-WA/apiDefinition.swagger.json]: @sriyen-msft Hello, sorry, just pushed a fix. (tested locally and via paccon

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@ridlees Please resolve the branch conflicts to get the PR merged

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**